docs/9-membership.md

```diff
@@ -151,154 +161,11 @@ For transparency purposes, we are disclosing each Guild member's team, the name/
  | TXRX | [Mikhail Kalinin](https://github.com/mkalinin) | 
 
 ## Addresses and Weights
-For member privacy, the order of weights below does not correspond to the names above. These addresses and weights will change over time as members change from full-time to part-time (or vice-versa), and as members are on/offboarded.
+Member addresses and weights can be seen in the [Split contract](https://app.0xsplits.xyz/accounts/0x84af3D5824F0390b9510440B6ABB5CC02BB68ea1/). For member privacy, the order of weights in the Split does not correspond to the names above.
```

Comment:
> For member privacy, the order of weights in the Split does not correspond to the names above.

this can be removed, it made more sense when there was a list of addresses in the docs